tests/test_fixtures/great_expectations_no_block_no_fluent_datasources.yml

```diff
@@ -0,0 +1,44 @@
+# This is a basic configuration for testing.
+# It has comments that should be preserved.
+config_version: 3
```

Comment:
Same question

Reply:
@Shinnnyshinshin I am preparing them for the next addition.  Thanks.